files/__addonLocation__/package.json

```diff
@@ -10,18 +10,25 @@
   "author": "",
   "files": [
     "addon-main.cjs",
-    "dist"
+    "dist",
+    "declarations"
```

Comment:
**suggestion**

Can we keep the list alphabetized?

```suggestion
    "declarations",
    "dist"
```

Reply:
🤷 sure. Any reason to prefer an order?